package benchmark.core.impl.exception;